rpmlb/custom.py

```diff
@@ -23,6 +24,7 @@ def run_cmds(self, key, **kwargs):
             return
         cmd_element = cmd_dict[key]
 
+        env['CUSTOM_DIR'] = os.path.dirname(os.path.abspath(str(self._file_path)))
```

Comment:
The alternative way of fixing this would be `str(self._file_path.resolve().parent)` and dropping the `os.path` import.

However, I wasn't sure if `self._file_path` was guaranteed to always be a `pathlib.Path` instance, so coercing first and then using the `os.path` functions seemed more robust.